test/js/sql/adapter-env-var-precedence.test.ts

```
@@ -0,0 +1,447 @@
+      password: "postgres",
+      port: 5432,
+    });
+
+    expect(options.options.adapter).toBe("postgres");
+    expect(options.options.hostname).toBe("bar_url");
+    expect(options.options.port).toBe(5432);
+    expect(options.options.username).toBe("postgres");
+  });
+
+  test("should only read PostgreSQL env vars when adapter is postgres", () => {
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)